bip-musig2/reference.py

```diff
@@ -278,7 +281,7 @@ def nonce_agg(pubnonces: List[bytes]) -> bytes:
 
 SessionContext = namedtuple('SessionContext', ['aggnonce', 'pubkeys', 'tweaks', 'is_xonly', 'msg'])
```

Comment:
Should `pubkeys` be renamed to `plain_pks` or something?

Reply:
I typed the tuple. Is that explicit enough?

Reply:
Yeah that works great, thanks!